packages/design-system/src/styles/components/_Dropdown.scss

```diff
@@ -18,4 +25,68 @@ select.ds-c-field {
     color: transparent;
     text-shadow: 0 0 0 #000;
   }
+
+  .ds-c-dropdown--open & {
+    border-end-end-radius: 0;
```

Comment:
This made me do a double-take 😆  I never knew that's how this radius was targeted!

Reply:
I started with "bottom-right" and then was like, "we're now using logical properties...how i do this?" It's pretty confusing, tbh. I wish I could say `border-end-radius` and do both at once.